tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "experimentalDecorators": true,
+    "importHelpers": false,
```

Comment:
This seems to be the default value. Was this intentionally included with the default value?

Reply:
`moduleResolution` and `removeComments` seem to be set to their default values as well

Reply:
Again, shamefully copied from GABA

Reply:
I'll remove these